iOS-Weather-App/Utils/AlertPopUp.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by pavels.garklavs on 02/03/2022.
+//
+
+import UIKit
+
+struct AlertPopUp {
+    static func makePopUp(controller: UIViewController, error: Error) {
```

Comment:
create an extension of UIViewController

extension UIViewController {
func showAlert(title: String, message: String, completion: @escaping () -> Void) {
let alert = ...
alert.addAction(..., handler: completion)
self.present(alert...)